homeassistant/components/overkiz/climate_entities/atlantic_heat_recovery_ventilation.py

```diff
@@ -0,0 +1,180 @@
+"""Support for AtlanticHeatRecoveryVentilation."""
+from __future__ import annotations
+
+from typing import cast
+
+from pyoverkiz.enums import OverkizCommand, OverkizCommandParam, OverkizState
+
+from homeassistant.components.climate import ClimateEntity
+from homeassistant.components.climate.const import (
+    FAN_AUTO,
+    ClimateEntityFeature,
+    HVACMode,
+)
+from homeassistant.const import TEMP_CELSIUS
+
+from ..coordinator import OverkizDataUpdateCoordinator
+from ..entity import OverkizEntity
+
+FAN_BOOST = "home_boost"
+FAN_KITCHEN = "kitchen_boost"
+FAN_AWAY = "away"
+FAN_BYPASS = "bypass_boost"
+
+PRESET_AUTO = "auto"
+PRESET_PROG = "prog"
+PRESET_MANUAL = "manual"
+
+OVERKIZ_TO_FAN_MODES: dict[str, str] = {
```

Comment:
Why explicitly give the typing here?

Reply:
This is required for mypy, otherwise it will think it needs `OverkizCommandParam` (enum), where we also pass a string.